connectivity/FEATURE_BLE/include/ble/Gap.h

```diff
@@ -431,8 +435,8 @@ class Gap {
          *
          * @version 4.1+.
          *
-         * @note This event is not generated if connection parameters update
-         * is managed by the middleware.
+         * @note This event will only be produced if manageConnectionParametersUpdateRequest(true)
```

Comment:
`manageConnectionParametersUpdateRequest(true)` won't be cross referenced

Reply:
Can you address this ? I think `This event will only be produced if manageConnectionParametersUpdateRequest() was called with true.` would work. What do you think ?

Reply:
OK, I misundestood, I though you meant the doc for manageConnectionParametersUpdateRequest isn't cross refed, so I added the @see over there.